frontend/js/src/explore/ExploreIndex.tsx

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import { createRoot } from "react-dom/client";
+import * as React from "react";
+import { get, has } from "lodash";
+import { getPageProps } from "../utils/utils";
+
+import ErrorBoundary from "../utils/ErrorBoundary";
+import GlobalAppContext, { GlobalAppContextT } from "../utils/GlobalAppContext";
+import SimpleModal from "../utils/SimpleModal";
+import APIServiceClass from "../utils/APIService";
+import BrainzPlayer from "../brainzplayer/BrainzPlayer";
+
+type ExplorePageProps = {
+  name: string;
+  desc: string;
+  img_name: string;
+  url: string;
+};
+
+function ExploreIndex(props: ExplorePageProps) {
```

Comment:
There's a small structural refactoring that I'd like here, to follow our style in the rest of the page components:

This component should be called something like `ExploreCard` with its `ExploreCardProps`, and you should create another component `ExploreIndex` or `ExplorePage` which contains what is currently at lines 68->121 (https://github.com/metabrainz/listenbrainz-server/pull/2372/files#diff-9de970bd4956e4d3e4f0a65bd19527cb4baea5250dd466c3325abbfbf520cbbaR68-R121)

You'll end up with this structure:
```
function ExploreCard(props: ExploreCardProps){
…
}

function ExplorePage(){
…
[this renders a bunch of ExploreCards]
}

[...]
renderRoot.render(
    <ErrorBoundary>
      <GlobalAppContext.Provider value={globalProps}>
        <ExplorePage />
      </GlobalAppContext.Provider>
    </ErrorBoundary>
)
```

That keeps the part in the DOMContentLoaded listener mostly separate from the implementation details of that page, allowing us to reuse that page component later on without the need to refactor it.